stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -683,9 +683,11 @@ dot(x::Number, y::Number) = conj(x) * y
     dot(x, y)
     x ⋅ y
 
-Compute the dot product between two vectors. For complex vectors, the first
-vector is conjugated. When the vectors have equal lengths, calling `dot` is
-semantically equivalent to `sum(dot(vx,vy) for (vx,vy) in zip(x, y))`.
+Compute the dot product between two arrays of the same size as if they were
```

Comment:
"of the same size" -> "with the same axes" ?  The "equal sizes" language is also used a few lines below...

Reply:
I will change that.